src/uct/ib/rc/base/rc_iface.h

```diff
@@ -489,8 +486,9 @@ static UCS_F_ALWAYS_INLINE void
 uct_rc_iface_add_cq_credits(uct_rc_iface_t *iface, uint16_t cq_credits)
 {
     iface->tx.cq_available += cq_credits;
-    ucs_assertv(iface->tx.cq_available <= iface->config.tx_cq_len,
-                "cq_available=%d tx_cq_len=%d cq_credits=%d",
+    ucs_assertv((ssize_t)iface->tx.cq_available <=
```

Comment:
maybe better cast tx_cq_len to int?

Reply:
if `tx_cq_len` > `IINT_MAX`, then `(int)tx_cq_len` will be less than `tx.cq_available`.
I think `tx_cq_len` > `IINT_MAX` is impossible, but I'd not cast to `int` to avoid potential bugs in the future or static checkers' warnings.

Reply:
okay